src/Utilities/Utilities.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Pinterest for WooCommerce Feed Logger
+ *
+ * @package     Pinterest_For_WooCommerce/Classes/
+ * @since       1.0.10
+ */
+
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\Pinterest\Utilities;
+
+use \Error;
+
+const PINTEREST_ACCOUNT_CONNECTION_TIMESTAMP = PINTEREST_FOR_WOOCOMMERCE_PREFIX . '-account-connection-timestamp';
+
+/**
+ * Set the account connection timestamp.
+ *
+ * @since x.x.x
+ * @throws Error Wrong value provided for the timestamp.
+ * @param int $timestamp Timestamp to set as the account connection moment.
+ * @return int
+ */
+function set_account_connection_timestamp( $timestamp = null ): int {
```

Comment:
Just curious, why isn't this a class instead of individual functions? We're already using `static` everywhere in this plugin, so this wouldn't be any different than the other classes.

Reply:
The intention is to have utils as a bag for all sorts of things. Cramming them all into a class is OK, but I thought that having them separate makes a bit more sense. This is a personal preference based on how I tend to think about how methods and data in class should relate to each other. I am happy to change this if you think that this is not good.